specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/storageaccounts.json

```diff
@@ -0,0 +1,593 @@
+        "x-ms-examples": {
+          "Returns the requested storage account.": {
+            "$ref": "./examples/StorageAccounts/Get.json"
+          }
+        },
+        "tags": [
+          "StorageAccounts"
+        ],
+        "operationId": "StorageAccounts_Get",
+        "description": "Returns the requested storage account.",
+        "parameters": [
+          {
+            "$ref": "storageaccounts.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "storageaccounts.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "storageaccounts.json#/parameters/AccountIdParameter"
+          },
+          {
+            "$ref": "storageaccounts.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The storage account has been returned.",
+            "schema": {
+              "$ref": "storageaccounts.json#/definitions/StorageAccount"
+            }
+          },
+          "404": {
+            "description": "NOT FOUND -- The storage account was not found."
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage.Admin/storageaccounts/{accountId}/undelete": {
+      "post": {
+        "x-ms-examples": {
+          "Undelete a deleted storage account.": {
+            "$ref": "./examples/StorageAccounts/Undelete.json"
+          }
+        },
+        "tags": [
+          "StorageAccounts"
+        ],
+        "operationId": "StorageAccounts_Undelete",
+        "description": "Undelete a deleted storage account.",
+        "parameters": [
+          {
+            "$ref": "storageaccounts.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "storageaccounts.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "storageaccounts.json#/parameters/AccountIdParameter"
+          },
+          {
+            "$ref": "storageaccounts.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "storageaccounts.json#/parameters/NewAccountNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Undelete either has been performed or account was not deleted.",
+            "schema": {
+              "$ref": "storageaccounts.json#/definitions/UndeleteStorageAccount"
+            }
+          },
+          "202": {
+            "description": "Accepted -- undelete operation trigged; operation will complete asynchronously."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage.Admin/storageaccounts/ondemandgc": {
```

Comment:
Can we call this 'start' instead of 'ondemand'? The way I read it now, it doesn't sound like an action. Typically actions start with a verb. Also, should we spell out GarbageCollection?

Reply:
We will change "ondemandgc" into "reclaimgarbagecollection"